src/Umbraco.Abstractions/Umbraco.Abstractions.csproj

```diff
@@ -15,4 +15,13 @@
       <PackageReference Include="System.Runtime.Caching" Version="4.6.0" />
     </ItemGroup>
 
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
Guess we need this after all :)

Reply:
For the test projects, I see no problem, but stuff should IMO be public between Abstractions and Core